.gitignore

```diff
@@ -3,6 +3,10 @@ __pycache__/
 *.py[cod]
 *$py.class
 
+# Datasets
+datasets/*.json
```

Comment:
Or the whole directory?

Reply:
I think just json for now.  But you make a good point.  We might want to exclude all ".json.bz2" as well.